README.md

```diff
@@ -117,326 +68,113 @@ https://pmi.pay1.de/
 
-### Build
+All these properties must be set for every tenant name described in `TENANTS` property above.
 
-The Integration is built as a "fat jar" that can be directly started via  the `java -jar` command. The jar is built as follows:
+Name                         | Content             | Required  |
+-----------------------------|---------------------|-----------|
+`TENANT1_PAYONE_PORTAL_ID`   | Payment portal ID   | **Yes**   |
+`TENANT1_PAYONE_KEY`         | Payment portal key  | **Yes**   |
+`TENANT1_PAYONE_MERCHANT_ID` | Merchant account ID | **Yes**   |
+`TENANT1_PAYONE_SUBACC_ID`   | Subaccount ID       | **Yes**   |
+`TENANT1_PAYONE_MODE`        | Payment mode        | No        |
 
-```
-./gradlew stage
-```
+Can be found in the [PAYONE Merchant Interface](https://pmi.pay1.de/).
 
-Run the JAR:
+##### Optional service configuration parameters
 
-```
-java -jar service/build/libs/commercetools-payone-integration.jar
-```
+All below are optional.
 
-### Run
+Name | Is tenant specific | Content | Default
+---- | ------- | ------ | --------
+`SHORT_TIME_FRAME_SCHEDULED_JOB_CRON` | No  | [QUARTZ cron expression](http://www.quartz-scheduler.org/documentation/quartz-1.x/tutorials/crontrigger) to specify when the service will poll for commercetools messages generated in the past 10 minutes like [PaymentInteractionAdded](http://dev.commercetools.com/http-api-projects-messages.html#payment-interaction-added-message) | poll every 30 seconds
+`LONG_TIME_FRAME_SCHEDULED_JOB_CRON`  | No  | [QUARTZ cron expression](http://www.quartz-scheduler.org/documentation/quartz-1.x/tutorials/crontrigger) to specify when the service will poll for commercetools messages generated in the past 2 days | poll every hour on 5th second
+`TENANT1_PAYONE_MODE`                 | Yes | the mode of operation with PAYONE <ul><li>`"live"` for production mode, (i.e. _actual payments_) or</li><li>`"test"` for test mode</li></ul> | `"test"`  
+`TENANT1_CT_START_FROM_SCRATCH`       | Yes | **WARNING** _**Handle with care!**_ If and only if equal, ignoring case, to `"true"` the service will create the custom types it needs. _**Therefor it first deletes all Order, Cart, Payment and Type entities**_. If not yet in the project, the Custom Types are created independently of this parameter (but only deleted and recreated if this parameter is set).  Related: [issue #34](https://github.com/commercetools/commercetools-payone-integration/issues/34). | `"false"`
+`TENANT1_SECURE_KEY`                  | Yes | if provided and not empty, the value is used as the key for decrypting data from fields "IBAN" and "BIC" for payments with CustomType "PAYMENT_BANK_TRANSFER". The data must be the result of a Blowfish ECB encryption with said key and encoded in HEX. | "" (empty String)
+`TENANT1_UPDATE_ORDER_PAYMENT_STATE`  | Yes | if _true_ - `Order#paymentState` will be updated when payment status notification is received from Payone. By default the order's state remains unchanged. | "false" 
```

Comment:
yes, in [Order Payment Status Mapping](https://github.com/commercetools/commercetools-payone-integration/blob/master/docs/Order-Payment-Status-Mapping.md) documentation.